### PR TITLE
DAT-4211 | add test for error page

### PR DESCRIPTION
**Pull request by tomasznapieralski**
@nikodamn

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/ErrorPageTests.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import com.wikia.webdriver.elements.mercury.pages.ErrorPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class ErrorPageTests extends NewTestTemplate {
+
+  @Test()
+  public void mercury_errorPage_navigateFromErrorPageToArticlePageUsingGlobalNav() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/ErrorPageTests.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/ErrorPageTests.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import com.wikia.webdriver.elements.mercury.pages.ErrorPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class ErrorPageTests extends NewTestTemplate {
+
+  @Test(groups = "mercury_errorPage_navigateFromErrorPageToArticlePageUsingGlobalNav")
+  public void mercury_errorPage_navigateFromErrorPageToArticlePageUsingGlobalNav() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/ErrorPageTests.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/ErrorPage.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Navigation;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ErrorPage extends WikiBasePageObject {
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/ErrorPage.java]: Please access the components present on the Error Page by lombok as described in the architecture doc:
https://wikia-inc.atlassian.net/wiki/display/AUT/How+to+build+a+Page+from+Components+using+Lombok

**Pull request merged**